galaxy_ng/tests/integration/api/test_collection_signing.py

```diff
@@ -149,6 +149,19 @@ def test_collection_auto_sign_on_approval(api_client, config, settings, flags, u
     assert metadata["signatures"][0]["signature"].startswith("-----BEGIN PGP SIGNATURE-----")
     assert metadata["signatures"][0]["pubkey_fingerprint"] is not None
 
+    # Assert that the collection is signed on UI API (detail endpoint)
+    collection_on_ui = api_client(
+        "/api/automation-hub/_ui/v1/repo/published/{namespace}/{collection}/?version=x.y.z"
```

Comment:
It is needed to interpolate the URL str with the variables
```suggestion
        f"/api/automation-hub/_ui/v1/repo/published/{NAMESPACE}/{artifact.name}/?version={artifact.version}"
```